draft-ietf-taps-transport-security.md

```diff
@@ -443,16 +443,19 @@ cannot decouple the security from transport functionality.
 # Application Interface {#application-interface}
 
 This section describes the interface surface exposed by the security protocols described above.
-Note that not all protocols support each interface. We partition these interfaces into
+We partition these interfaces into
 pre-connection (configuration), connection, and post-connection interfaces, following
 conventions in {{?I-D.ietf-taps-interface}} and {{?I-D.ietf-taps-arch}}.
 
+Note that not all protocols support each interface.
+The table in {{interface-protocols-table}} summarizes which protocol exposes which of the interfaces.
+
 ## Pre-Connection Interfaces
 
 Configuration interfaces are used to configure the security protocols before a
 handshake begins or the keys are negotiated.
```

Comment:
It may be good to mention that the abbreviations used here are just keys for the table, not "official" acronyms or anything.

Reply:
Added a sentence that explains why we provide abbreviations.
If anyone wants to phrase this in a different way, I'm open to suggestions.